memberships/views.py

```diff
@@ -141,3 +144,45 @@ def stripe_webhook(request):
         return stripe_webhook.handle(event)
     except ValueError as e:
         return HttpResponse("Failed to parse stripe payload", status=400)
+
+
+@login_required()
+def details_view(request):
```

Comment:
I'm not 100% sure what this does. I thought the details_view just rendered the current information we had available for a member, but this also deals with POST requests?

What is the POST request here supposed to do?

Reply:
The member details page is a modelform just like the member settings page is. We have two different views so that we can exclude different things on each page. I have tried removing lines 151:154 but then the member details page does not display the data. We could build the member details page through several queries, however, as I understand it, we would then lose out on the automagicness of modelforms.

Reply:
I guess what I am trying to say is, why can't this view just be this:

```py
@login_required()
def details_view(request):
	return render(request, "memberships/member_details.html", {
		"form": MemberDetailsForm(instance=request.user.member,
		"profile_image": request.user.member.profile_image
	});
```

As I'm guessing we do not expect the member details form to be submitted, so we are only expecting to response to `GET` requests.

Reply:
Fair cop! I thought the other stuff was necessary for the form, turns out its not. That works now, thanks mate.